### PR TITLE
Add wakatime layer

### PR DESCRIPTION
**Pull request by CestDiego**

### CODE REVIEW

File: contrib/!tools/wakatime/packages.el [hunk 1]
```diff
@@ -0,0 +1,9 @@
+(defvar wakatime-packages
```
Comment (trishume) [inline contrib/!tools/wakatime/packages.el]: Should be `setq` for reloading support.

**Comment by CestDiego** [comment]
@trishume sorry I think I had an old template :P

**Comment by trishume** [comment]
I was going to say that this should be an additional packages thing because of the small layer elisp, but I realized that the real value here is the great Readme.

File: contrib/!tools/wakatime/packages.el [hunk 2]
```diff
@@ -0,0 +1,9 @@
+(setq wakatime-packages
+  '(
+    wakatime-mode
+    ))
+
+(defun wakatime/init-wakatime-mode ()
+  (use-package wakatime-mode
+    :init
+    (add-hook 'prog-mode-hook 'global-wakatime-mode)))
```
Comment (syl20bnr) [inline contrib/!tools/wakatime/packages.el]: Why `global` ? If you want it global then the hook is not required.

Comment (CestDiego) [inline contrib/!tools/wakatime/packages.el]: oh...my bad

Comment (CestDiego) [inline contrib/!tools/wakatime/packages.el]: changed

**Comment by syl20bnr** [comment]
Thnak you :+1: 
Cherry-picked into `develop` branch, you can safely delete your branch.

**Pull request closed without merging**